examples/notebooks/beam-ml/mltransform/compute_and_apply_vocab.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
```

Comment:
Instead of `examples/notebooks/beam-ml/mltransform`, what do you think of `examples/notebooks/beam-ml/data-prep` or something in that vein? I don't think its too important, but it might be more discoverable if we go with a use case driven naming convention

Reply:
Sounds good. I will make the change before merging.